<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnConnect">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMoveToStation">
      <property name="text">
       <string>Move to station</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRotate">
      <property name="text">
       <string>Rotate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPing">
      <property name="text">
       <string>Ping</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAskForReport">
      <property name="text">
       <string>Ask for report</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="teConsole"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leCmdInput"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>339</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
   </widget>
   <addaction name="menuOption"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
